AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/InitialSync/InitialSyncOperation.swift

```diff
@@ -43,34 +51,31 @@ final class InitialSyncOperation: AsynchronousOperation {
         }
 
         log.info("Beginning sync for \(modelType.modelName)")
-        setUpLastSyncTime()
-        query()
+        let lastSyncTime = getLastSyncTime()
+        query(lastSyncTime: lastSyncTime)
     }
 
-    private func setUpLastSyncTime() {
+    private func getLastSyncTime() -> Int? {
         guard !isCancelled else {
             super.finish()
-            return
+            return nil
         }
 
         let lastSyncMetadata = getLastSyncMetadata()
         guard let lastSync = lastSyncMetadata?.lastSync else {
-            return
+            return nil
         }
 
         //TODO: Update to use TimeInterval.milliseconds when it is pushed to master branch
```

Comment:
Consider adding an item to our backlog & linking it here so we don't lose track of this TODO

Reply:
done.
https://github.com/aws-amplify/amplify-ios/issues/398